tests/rptest/tests/tiered_storage_model_test.py

```diff
@@ -476,8 +508,16 @@ def run_stage_validators(self, stage, test_case):
                 self.logger.info(
                     f"{stage} delayed by lagging validators: {lagging_validators}"
                 )
-                time.sleep(2 * ix)
-        assert done == total, f"Failed to validate data after {MAX_RETRIES} runs"
+                failed_validators = lagging_validators
+                if ix % 2 == 0:
+                    # Clean up cache on all nodes to force segment hydrations
+                    time.sleep(60)  # Wait STM max eviction time
```

Comment:
nit: is this configured somewhere in this test? Can it be a constant so it's easy to figure out what to tune if needed?

Also it's not clear, is this sleep meant to wait for some other condition (e.g. eviction to happen), in which case should we just follow the manifest start offset to check for eviction?

Reply:
it's set to the same value as eviction timeout in `materialized_resources`, currently this value is hardcoded in C++